specification.md

```diff
@@ -0,0 +1,152 @@
+# Specification
+
+This specification provides a mechanism for projects to report information about their security in a machine-processable way. It is formatted as a YAML file to make it easy to read and edit by humans.
+
+Values that are included within the specification may be required or optional. Optional values are reccommendations from the Open Source Security Foundation's _Identifying Security Threats Working Group_, but may not be prudent for all use cases.
+
+The following values are part of Security Insights Specification `v1.0.0`
+
+## Header
+Here is an example of the header section, which holds high-level information about the project and its security documentation.
+
+```yaml
+header:
+  schema-version: 1.0.0
+  last-updated: '2021-09-01'
+  last-reviewed: '2022-09-01'
+  expiration-date: '2023-08-31T10:10:09.000Z'
+  project-url: https://github.com/foo/bar
+  commit-hash: 4dbf78ebc006ee5f668c0a74876ef8d6db9485be
+  project-release: '1.2.0'
+  changelog: https://github.com/foo/changelog.md
+  license: https://git.foo/license
+  parent-security-yaml: https://blah.com/ossf-security.yaml
+```
+
+Now let’s break that down:
+- `schema-version` (Required): Provide the version of the specification that you are adhering to.
+  - **String**
+- `last-updated` (Required): Date of the most recent change to this file.
+  - **Date**
+- `last-reviewed` (Required): Date this file was last reviewed to confirm holistic accuracy.
+  - **Date**
+- `expiration-date` (Required): Date this file should no longer be considered valid. Maximum one year from the date it was last reviewed.
+  - **Date-time**
+- `project-url` (Required): URI for the project repository that this document pertains to.
+  - **String** beginning with **https://**
+- `commit-hash` (Required): The last commit on the repository that this document pertains to.
+  - **String** matching the git SHA format
+- `changelog`: URI for the changelog for the project.
+  - **String** beginning with **https://**
+- `license`: URI for the license for the corresponding version of the project.
+  - **String** beginning with **https://**
+- `parent-security-yaml`: If the document is mirrored from another, give a URI pointing to the original.
+  - **String** beginning with **https://**
+
+## Project Lifecycle
+The project-lifecycle section contains information about how the project is maintained. This is the first of many places where we’ll see conditional requirements in the schema.
+
+```yaml
+project-lifecycle:
+  stage: active
+  roadmap: https://foo.bar/roadmap.html
+  bug-fixes-only: false
+  core-maintainers:
+    - https://github.com/scovetta
+    - joe.bob@email.com
+```
+
+Note that the last value on this list is a conditional requirement, making it optional if the project is no longer being maintained. We’ll see more values like this in the following sections.
+- `stage` (Required): The project’s maintenance stage.
+  - **String**. Values may be active or inactive.
```

Comment:
Project maintainers may need more nuanced options than active/inactive, but that's a discussion for the spec. [Repostatus](https://www.repostatus.org/) thinks about this more.

Reply:
@luigigubello @scovetta What is the current perceived value of `stage` here? Would we be better served by changing this to `repostatus`?